eZ/Publish/Core/Persistence/Legacy/Tests/Content/Location/Gateway/DoctrineDatabaseTrashTest.php

```diff
@@ -400,6 +400,56 @@ function ($trashItem) {
         );
     }
 
+    /**
+     * @covers \eZ\Publish\Core\Persistence\Legacy\Content\Location\Gateway\DoctrineDatabase::listTrashed
+     */
+    public function testListTrashSortedTrashedDate()
```

Comment:
Please create API integration test instead. 
The tests in the Core namespace are run on SQLite only, so, given the known issues with PostgreSQL ordering, are not the most useful (and I'm gonna move them to API anyway once I find some time).

Reply:
ok, may I remove `testListTrashSortedTrashedDate`  or keep both?

Reply:
ok, Core test reimplemented as API integration.